### PR TITLE
Add stub for queue.SimpleQueue in 3.7

### PR DESCRIPTION
**Pull request by nabnut**

### CODE REVIEW

File: stdlib/3.7/simplequeue.pyi [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from typing import TypeVar, Generic, Optional
```
Comment (JelleZijlstra) [inline stdlib/3.7/simplequeue.pyi]: Seems like this should be in queue.pyi instead. You can use an `if sys.version_info ...` block to make this class accessible only in 3.7.

Comment (nabnut) [reply stdlib/3.7/simplequeue.pyi]: Thanks, I wasn't sure. Fixed.

**Comment by nabnut** [comment]
Anything missing for this to get merged ? thanks

**Comment by nabnut** [comment]
Noted, thanks for the advice :)

**Pull request merged**